pkg/apis/infinispan/v1/types_util.go

```diff
@@ -196,6 +196,10 @@ func (ispn *Infinispan) IsUpgradeCondition() bool {
 	return ispn.IsConditionTrue(ConditionUpgrade)
 }
 
+func (ispn *Infinispan) GetServiceFqn() string {
```

Comment:
Not sure we need Service FQN for connection between Job Pod and Infinispan Pod's

Reply:
Good point. The Batch  CR always has to reside in the same namespace as the target cluster, so the svc name alone should suffice.